tests/integration/test_fim/test_basic_usage/test_basic_usage_deferred_delete_folder.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+from subprocess import Popen, PIPE, DEVNULL
+import re
+import json
+from json import JSONDecodeError
+
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, create_file, REGULAR, \
+    callback_detect_event, check_time_travel, validate_event

```

Comment:
Please, remove unused imported functions: `check_time_travel` and `validate_event`.